tests/repository_tests.rs

```diff
@@ -0,0 +1,211 @@
+// Write your tests here.
```

Comment:
+ asi by som si premenoval funkcie na `pub fn test_....` a nazov funkcie,
+ nazov suboru na `repositories` (tak ako nazov priecinku)

ale to sú len také konvencie 😄